intermediate_source/torch_export_aoti_python.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+"""
+torch.export AOT Inductor Tutorial for Python runtime
+===================================================
+**Author:** Ankith Gunapal
+"""
+
+######################################################################
+#
+# .. warning::
+#
+#     ``torch._export.aot_compile`` and ``torch._export.aot_load`` are in Beta status and are subject to backwards compatibility
+#     breaking changes. This tutorial provides an example of how to use these APIs for model deployment using Python runtime.
+#
+# It has been shown `previously <https://pytorch.org/docs/stable/torch.compiler_aot_inductor.html#>`__ how AOTInductor can be used 
+# to do Ahead-of-Time compilation of PyTorch exported models by creating
+# a shared library that can be run in a non-Python environment.
+#
+#
+# In this tutorial, you will learn an end-to-end example of how to use AOTInductor for python runtime.
+# We will look at how  to use :func:`torch._export.aot_compile` to generate a shared library.
+# We also look at how we can run the shared library in python runtime using :func:`torch._export.aot_load`.
+#
+# **Contents**
+#
+# .. contents::
+#     :local:
+
+
+######################################################################
+# Model Compilation
+# ------------
+#
```

Comment:
@agunapal thanks for the PR, it would be also helpful to add the value prop of aot_compile, also if we have any numbers to better showcase why ppl should use it.

Reply:
this is the same comment as @desertfire mentioned above, not sure if its resolved.